src/figuresClasses.ts

```diff
@@ -1,11 +1,85 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    private sideA: number,
+    private sideB: number,
+    private sideC: number,
+  ) {
+    if (sideA <= 0 || sideB <= 0 || sideC <= 0) {
```

Comment:
you can create function like
```
function checkOnError(errorMessage ,...args: number[]): void {
  if (args.some((arg: number) => arg <= 0)) {
    throw new Error(errorMessage);
  }
}
```
It is needed because you have the same logic in three places.